src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -127,10 +126,12 @@ public Hardware() {
                 this.rightIntakeColorSensor = new MultiplexedColorSensor(RIGHT_INTAKE_COLORSENSOR_PORT);
                 this.centerIntakeColorSensor = new MultiplexedColorSensor(CENTER_INTAKE_COLORSENSOR_PORT);
             }
-
         }
         if (INDEX_ENABLED) {
-            indexMotor = new WPI_TalonFX(INDEX);
+            firstIndexMotor = new WPI_TalonFX(INDEX_1);
```

Comment:
Consider renaming index 1 and 2 to match the subsystem so there is no confusion.